app/views/orders/show.html.erb

```diff
@@ -34,13 +34,10 @@
   <% @order.item_orders.each do |item_order|%>
     <tr>
     <section id = "item-<%=item_order.item_id%>">
-        <td><p><% if item_order.fulfilled? %></p></td>
-            <td><p>"Fulfilled"</p></td>
-        <td><p><% else %></p></td>
-            <td><p>"Unfulfilled"</p></td>
-        <td><p><% end %></p></td>
-        <td><p><%=link_to item_order.item.name, "/items/#{item_order.item_id}"%></p></td>
-        <td><p><%= link_to item_order.item.merchant.name, "/merchants/#{item_order.item.merchant.id}"%></p></td>
+        <td><p><%= "Fulfilled" if item_order.fulfilled? %></p></td>
+        <td><p><%= "Unfulfilled" unless item_order.fulfilled? %> </p></td>
```

Comment:
Nice! we could also further trim down the code like
`<td><p><%= item_order.fulfilled? ? "Fulfilled" : "Unfulfilled" %></p></td>`